blog/2023-03-17-project-coordinator.md

```diff
@@ -0,0 +1,38 @@
+---
+author: Antonin Delpeuch
+title: "OpenRefine is looking for a part time project coordinator"
+---
+
+OpenRefine is looking for a part time project coordinator.
+This paid role is intended to help facilitate community processes in the project, improve our capacity to liaise with partners and our governance structures.
+
+The responsibilities of this project coordinator will include:
+* Facilitating regular meetings within the OpenRefine community and with external partners;
+* Run outreach activities such as our biyearly user survey or by representing the project in events;
+* Maintain the project's communication channels (such as moderating the forum)
+* Help organize in-person events for the OpenRefine community
+* Coordinate our participation in internship programmes such as Outreachy or Google Summer of Code;
+* Help with grant applications and reporting
+* Liaise with Code for Science and Society, our fiscal sponsor
+
+The scope of the role is anticipated to evolve, to adapt to the coordinator's interests and the project's needs.
+
+Requirements:
+* Good communication skills and fluency in English, both written and spoken
+* Self-motivated and organized
+* Able to work fully remotely
+
+Nice to have:
+* Experience with working in a volunteer organization
+* Familiarity with OpenRefine or a related open source project
```

Comment:
I don't see how this is optional if they are going to be representing the OpenRefine project at events or effectively communicating with the community.

Reply:
Happy to make it a requirement. My reasoning was that because this is a support role I thought it could likely work without: basically, anyone well organized and enjoying coordination tasks could fit in my opinion.
I think we are likely to have enough people who tick that box anyway.

See also this similar posting from the Inkscape project which I took some inspiration from:
https://web.archive.org/web/20220603054003/https://inkscape.org/jobs/project-coordinator-2022/